### PR TITLE
Portfolio: Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by VivekRaaZ**
Project requirements
Build the End section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:

"Works" section✅

The "Contact me" section (placed after the projects "About myself"): Contact me✅

Make sure that:
Name field accepts up to 30 chars.✅
Email field accepts only emails. ✅
Message field accepts up to 500 chars.✅
All fields have placeholder text. ✅
A user cannot submit the form if one or more of the fields is empty. ✅
In order to lay out the elements on the page you should use flexbox. ✅
You don't need to implement any functionality that requires JavaScript. ✅
Implement the button interactions (enable, hover, etc.). ✅
The website should be responsive, following these breakpoints:
Minimum mobile screen size should be 375px✅
Minimum desktop screen size should be 768px✅
Hope you like it! 😃👍

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @VivekRaaZ, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a pair programming project with no contributions from your partner.  Kindly report any issue you had with your partner to the `student success` team and they'll permit you to work solo or look for a suitable partner. Attach a screenshot of the message you received from `student success` stating that you can do the project alone in the pull request description and resubmit this PR👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -245,6 +245,31 @@ <h5 class="skill">
...
+    <!-- ---------- Contact me ---------- -->
+
+    <section class="myself">
+      <img src="image/contact form.png" alt="contact">
+      <aside class="touch">
+      <h2 class="last">Contact me</h2>
+      <p class="end">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+      
+        <form class="form1">
```
Comment (Ibizugbe) [inline index.html]: Nice job making your project match the given design😉. just as required for this milestone, kindly ensure your form makes use of the formspre service to retrieve data from clients😉.

image

File: README.md [hunk 2]
```diff
@@ -122,14 +122,27 @@ You can deploy this project using: github, VS Code
...
 - Twitter: [@twitterhandle](https://twitter.com/BossVivekRaj)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/vivek-raj-a78b92256/)
 
+  👤 **Author 2**
+
+👤 **Okeke Okechukwu**
+
+- GitHub: [@githubhandle](https://github.com/okayboy12)
+- Twitter: [@twitterhandle](https://twitter.com/okayboy)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okayboy)
```
Comment (Ibizugbe) [inline README.md]: Nice job updating your readme file😉. Currently, the link to your `mit` license is broken. kindly do well to fix it as this is giving your project a less professional look👍🏼.

image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @VivekRaaZ 👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Project matches the given design👍🏼.
- [x] `flexbox` is used to layout elements in the contact section👍🏼.
- [x] The required form validations has been added to each input field👍🏼.
- [x] Linters are passing👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @okayboy12, @VivekRaaZ , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

Dear @okayboy12 and @VivekRaaZ, Thank you both for submitting your projects. However, we have noticed that the current code review request is invalid. This particular Pull Request is intended for @VivekRaaZ, and he should submit the review request from his own dashboard. 
- Additionally, @okayboy12, you are required to submit your project for review separately, also from your own project's dashboard. Even though both of you are working on the same project and milestone, each of you needs to request a review for your own portfolio project.
- To clarify, both of you should work on both of your portfolios for the same milestone, but you have to request reviews from your individual portfolio projects. Please make the necessary adjustments, and once you are ready, kindly resubmit your respective code review requests from your individual portfolios. Please avoid opening new Pull Requests for re-reviews; use the original requests unless instructed otherwise.

## 🔴🚫Attention🚫🔴 

📢 Your Code Review Request will be marked as invalid🚫 in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

## ⚠️ WARNING ⚠️

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_**📢💡Invalid Code Review Request **does not count** into the code reviews limit.💡**_

**Review by V-Blaze — APPROVED** [review]
Hi @VivekRaaZ  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- UI  looks great💯
-  Integrated formspree successfully👍
-  Clean and well documented code🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**